### ISSUE TITLE
[Bug]: Address search with nominatim.openstreetmap.org no longer working, API change

### ISSUE THREAD

**Issue by fl0e** [nextcloud/maps#1111]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Maps app **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Searching for a city  returns 404 error from nominatim.openstreetmap.org visible in browsers developer tools (console and network tab).

Current syntax used with the example searching for "Berlin":
https://nominatim.openstreetmap.org/search/Berlin?format=json&addressdetails=1&extratags=1&namedetails=1&limit=5
Opening this URL in the browser shows a 404 error page explaining that the syntax for searching changed, excerpt:
> Using the URL /search/ and /reverse/ (with slashes) is no longer supported.
> You use: https://nominatim.openstreetmap.org/search/US/Texas/Huston
> Change to: https://nominatim.openstreetmap.org/search?q=Huston, Texas, US

When I change the URL as instructed in the browser to https://nominatim.openstreetmap.org/search?q=Berlin&format=json&addressdetails=1&extratags=1&namedetails=1&limit=5 I get a json array of objects as response, so probably more as expected, excerpt
```
[{
  "place_id":307789109,
  "licence":"Data © OpenStreetMap contributors, ODbL 1.0. http://osm.org/copyright",
  "osm_type":"relation",
  "osm_id":62422,
  "lat":"52.5170365",
  "lon":"13.3888599",
  ..
  "addresstype":"city",
  "name":"Berlin",
  "display_name":"Berlin, Deutschland",
}]

```

So a fix might be as easy as changing the syntax for this search

### Steps to reproduce

1. Search for a city, i.e. Berlin
2. No results for addresses are returned (only results in contacts and other local data)

### Expected behavior

Searching for a city or address returns a list of results as it was until recently

### Installation method

Manual installation

### Operating system

Debian/Ubuntu

### PHP engine version

Other

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install of the server?

None

### Is this bug present after an update or on a fresh install of the app?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [X] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
Enabled:
  - activity: 2.19.0
  - announcementcenter: 6.6.1
  - api_server_monitor: 0.1.2
  - calendar: 4.4.4
  - circles: 27.0.1
  - cloud_federation_api: 1.10.0
  - comments: 1.17.0
  - contacts: 5.3.2
  - contactsinteraction: 1.8.0
  - dav: 1.27.0
  - deck: 1.10.0
  - drawio: 2.1.2
  - federatedfilesharing: 1.17.0
  - federation: 1.17.0
  - files: 1.22.0
  - files_pdfviewer: 2.8.0
  - files_rightclick: 1.6.0
  - files_sharing: 1.19.0
  - files_trashbin: 1.17.0
  - files_versions: 1.20.0
  - firstrunwizard: 2.16.0
  - forms: 3.3.1
  - gpoddersync: 3.8.1
  - gpxmotion: 0.1.0
  - gpxpod: 5.0.12
  - groupfolders: 15.0.1
  - jitsi: 0.17.0
  - logreader: 2.12.0
  - lookup_server_connector: 1.15.0
  - maps: 1.1.0
  - metadata: 0.19.0
  - news: 22.0.0
  - nextcloud_announcements: 1.16.0
  - notes: 4.8.1
  - notifications: 2.15.0
  - oauth2: 1.15.1
  - oidc_login: 2.6.0
  - password_policy: 1.17.0
  - photos: 2.3.0
  - previewgenerator: 5.3.0
  - privacy: 1.11.0
  - provisioning_api: 1.17.0
  - related_resources: 1.2.0
  - richdocuments: 8.1.1
  - serverinfo: 1.17.0
  - settings: 1.9.0
  - sharebymail: 1.17.0
  - snappymail: 2.28.4
  - spreed: 17.0.3
  - support: 1.10.0
  - survey_client: 1.15.0
  - systemtags: 1.17.0
  - text: 3.8.0
  - theming: 2.2.0
  - twofactor_backupcodes: 1.16.0
  - updatenotification: 1.17.0
  - user_status: 1.7.0
  - viewer: 2.1.0
  - weather_status: 1.7.0
  - workflowengine: 2.9.0
```

### Browser Console

```shell
https://nominatim.openstreetmap.org/search/Berlin?format=json&addressdetails=1&extratags=1&namedetails=1&limit=5
Status 404
Not Found
Version HTTP/2
Übertragen 944 B (780 B Größe)
Referrer Policyno-referrer
```

### Additional info

Since the cause for this issue is apparently a change in the external API of openstreetmap.org I will for now not provide all the details like nextcloud logs and such.

**Comment by geonom** [comment]
Having the same issue - here is the github issue which is linked in the error message:

https://github.com/osm-search/Nominatim/issues/3134

**Comment by tcitworld** [comment]
Closed by https://github.com/nextcloud/maps/pull/1110 ?

**Comment by fl0e** [comment]
Hi @tcitworld and thank you for hinting on #1110.
I pulled current master and build it. For me the search works again. I did not run any intensive tests but a couple of manual searches for addresses and POIs worked as expected.

> Closed by #1110 ?
therefore yes
